java/ql/lib/semmle/code/java/frameworks/MyBatis.qll

```diff
@@ -102,3 +102,40 @@ class MyBatisSqlOperationAnnotationMethod extends Method {
 class TypeParam extends Interface {
   TypeParam() { this.hasQualifiedName("org.apache.ibatis.annotations", "Param") }
 }
+
+module ProviderInjection {
```

Comment:
What's the reason for creating this module?

Reply:
Importing just the needed stuff in OGNL and SQL Injection queries :)

Reply:
I believe my last comment is wrong, since the whole framework is already imported while using some other libraries like `ExternalFlow`, and so making a module would not impact the queries at all (?).

Reply:
Yes, I don't think the module is necessary.